docs/reference/field-ref.asciidoc

```diff
@@ -2,32 +2,45 @@
 [role="xpack"]
 == Elastic Security ECS field reference
 
-This section lists ECS fields Elastic Security uses to display data.
+This section lists {ecs-ref}[Elastic Common Schema] (ECS) fields used by {elastic-sec} to provide an optimal SIEM and security analytics experience to users.  These fields are used to display data, provide rule previews, enable detection by prebuilt detection rules, provide context during rule triage and investigation, escalate to cases, and more.
 
-IMPORTANT: It is recommended to use {beats} to ship your data. Beat modules
-(for example, {filebeat-ref}/filebeat-modules.html[{filebeat} modules])
-are ECS-compliant so manual field mapping is not required, and all {es-sec-app}
-data is automatically populated. If you cannot use {beats}, map your data to
-{ecs-ref}[ECS fields] (see {ecs-ref}/ecs-converting.html[how to map data to ECS]).
-You can also try using the experimental https://github.com/elastic/ecs-mapper[ECS Mapper] tool.
+IMPORTANT: We recommend you use {beats} or {agent} integrations to ship your data to {es-sec}. Beat modules (for example, {filebeat-ref}/filebeat-modules.html[{filebeat} modules]) and {agent} integrations are ECS-compliant, which means data they ship to {es-sec} will automatically populate the relevant ECS fields.
```

Comment:
Seeing that we've been trying to promote Fleet more, it might be better to mention agent integrations first and then Beats. WDYT?

Reply:
++